### DISCUSSION TITLE
Compatibility with WP All Import

### DISCUSSION THREAD

**Discussion by sirjoshua** [pods-framework/pods#6699] [Ideas]
WP All Import, an advanced import tool for Posts/CPT/Tax/etc...
WP All Import
Import XML / CSV
http://www.wpallimport.com/

The specific issue i faced earlier today was:
- Custom Post Type POD with a custom field set to required

Upon import, the POD interference would always cause the import to terminate because the field "is required". Please note all data for these fields were not empty. 

Upon changing the required option in pods fields to be no, the import worked.

would like to see this fixed one way or another, was hard to debug.

**Comment by lukecav** [comment]
A good person to reach out to in relation to WP All Import is.
https://github.com/joeguilmette

**Comment by quasel** [comment]
related to #4586 - is this still an issue?
